### DISCUSSION TITLE
Smooth scrolling

### DISCUSSION THREAD

**Discussion by Ruttie2006** [AvaloniaUI/Avalonia#9917] [Q&A] [answered]
Is there a way to get things like ListBox or ScrollViewer to use smooth scrolling?
I don't really like the current feel of them 😅
(It's fine if only the nightly build has them)

**Comment by danielmayost** [comment, marked as answer]
#9677
still in working, i think it's nearing completion.

**Reply by DefrimBinakaj** [reply]
any progress? I'm looking to utilize this

**Reply by stevemonaco** [reply]
Smooth scrolling is not built-in for mouse scrolling. It only works with gestures. See https://github.com/AvaloniaUI/Avalonia/issues/12977 or look at SukiUI for an approximation which should work for most basic scenarios, especially those without virtualization.

**Reply by Egolds** [reply]
> any progress? I'm looking to utilize this

Check this:
https://github.com/Egolds/Avalonia.Xaml.Interactions.Animated
